pygotham/frontend/talks.py

```diff
@@ -100,6 +100,13 @@ def proposal(pk=None):
         db.session.add(talk)
         db.session.commit()
 
+        speaker = Speaker(
```

Comment:
Do we want to add the primary speaker to the speakers table? If so, we probably want to remove some fields from the talks table (which will require a data migration). We'll also need some way of tracking the primary speaker.